rust-toolchain

```diff
@@ -1 +1 @@
-1.66
+stable
```

Comment:
In the past we explicitly set the rust version in the rust-toolchain to cope with backward compatibility in future releases. I.e., at any point in time we exactly know which Rust version zenoh-c binding can be compiled with. So, I'd say to better bump the rust-toolchain to `1.67` instead.